enterprise/frontend/src/embedding-sdk/hooks/private/use-load-question.ts

```diff
@@ -22,9 +22,6 @@ type LoadQuestionResult = Promise<
 >;
 
 export interface LoadQuestionHookResult {
-  // The ID provided to the question component.
-  originalId?: number | string;
-
```

Comment:
We never returned this in the `useLoadQuestion` hook. The type is wrong, but it was incorrectly used as a union in the InteractiveQuestion types.